beacon_node/network/src/sync/block_lookups/common.rs

```diff
@@ -371,27 +371,35 @@ impl<L: Lookup, T: BeaconChainTypes> RequestState<L, T> for BlobRequestState<L,
 
     fn verify_response_inner(
         &mut self,
-        _expected_block_root: Hash256,
+        expected_block_root: Hash256,
         blob: Option<Self::ResponseType>,
         peer_id: PeerId,
     ) -> Result<Option<FixedBlobSidecarList<T::EthSpec>>, LookupVerifyError> {
         match blob {
             Some(blob) => {
                 let received_id = blob.id();
+
                 if !self.requested_ids.contains(&received_id) {
-                    self.state.register_failure_downloading();
                     Err(LookupVerifyError::UnrequestedBlobId)
+                } else if blob.verify_blob_sidecar_inclusion_proof().unwrap_or(false) {
```

Comment:
Ah this should be 
```suggestion
                } else if !blob.verify_blob_sidecar_inclusion_proof().unwrap_or(false) {
```

This is what's causing the tests to fail